src/components/List.jsx

```diff
@@ -78,7 +87,13 @@ export const List = ({ token }) => {
                     onChange={(e) => handleClick(doc, e)}
                   />
                   <label htmlFor={doc.id}>{doc.data().item}</label>
-                  {doc.data().total_purchases > 0 && (
+                  <button
+                    className="delete-button"
+                    onClick={() => deleteItem(doc)}
+                  >
+                    delete
+                  </button>
+                  {/* {doc.data().total_purchases > 0 && (
```

Comment:
looks like we're commenting some stuff out here, is it safe to just remove it?

Reply:
Almost. The other team was using it this week.